### ISSUE TITLE
Favicon

### ISSUE THREAD

**Issue by joostdevries** [alexspeller/ember-twiddle#44]

**Comment by Gaurav0** [comment]
How about this:

![favicon](https://cloud.githubusercontent.com/assets/313960/8893259/3f07167c-3358-11e5-8ee1-37f86bd6e730.png)

**Comment by Gaurav0** [comment]
Alternatively, we can commission a tomster like http://builtwithember.io

http://emberjs.com/tomster/faq/

**Comment by acorncom** [comment]
@wifelette Would it be possible to us to "re-use" one of these Tomsters or are they strictly "verboten"? :-)

![Built With](http://emberjs.com/images/tomsters/builtwith.png)
![Construction](http://emberjs.com/images/tomsters/construction.png)

Seems like they're probably off limits, but thought it might be worth confirming ...

**Comment by Gaurav0** [comment]
@acorncom @wifelette If it is possible I would prefer to commission a new Tomster. I would like to use it not just for the favicon, but for a splash screen. And I'd like to even get stickers to pass around if possible. :)

**Comment by acorncom** [comment]
I like the idea of using it for the splash screen :-)

**Comment by joostdevries** [comment]
> I would like to use it not just for the favicon, but for a splash screen. And I'd like to even get stickers to pass around if possible. :)
:heart_eyes:  now we just need someone who can actually create one. @rwjblue do you know whom we might ask such a thing

**Comment by rwjblue** [comment]
Per the [tomster faq](http://emberjs.com/tomster/faq/), I believe we can use the newly created [commision form](http://emberjs.com/tomster/commission/) to get the ball rolling.

**Comment by Gaurav0** [comment]
Filled out.

**Comment by Gaurav0** [comment]
Our request for a tomster has not been approved. Furthermore, we are not permitted to use the Ember logo or anything else trademarked by Ember for now.

**Comment by wifelette** [comment]
To clarify: the other day I received a request for approval on a logo someone else had created incorporating the Ember logo in a way that specifically violated the terms listed on the Logos page on the Ember website (by modifying the project logo / incorporating it in another logo). So turning it down wasn't related to Ember Twiddle in particular. I try to do my best to make sure people don't put effort into things that won't work by clarifying the language on the Logos page any time things comes up, and also by encouraging people to ask before they even work on anything. Always getting better!

As for the Tomster, we haven't yet reached consensus on a plan for Tomsters for subprojects or anything other than the specific use cases we already have existing Tomsters for, and we also know that we've made some mistakes in the past that we want to carefully avoid in the future. The Tomster and our brand are things we take _very_ seriously; the Tomster is among Ember's most precious assets, and by being super careful we make sure it stays that way <3

The fact that we have different kinds of projects and subprojects makes it challenging to find consistent policies that strengthen the brand. It's a high priority for us, but we also don't want to rush into anything. I'm sure we'll get it right in time :smile: 

What do people think about having the project (eventually) live at twiddle.emberjs.com? I know @wycats and @Gaurav0 have been talking about tweaks/fixes to Twiddle to make it more part of the Ember project and he'll be opening some issues about it soon; personally, I like clarity about what's official and what isn't, and this seems like one of the things that can help with that. I'll bring this up with @wycats and the rest of the team on the next call.

**Comment by Gaurav0** [comment]
@wifelette , I have no interest in denigrating the Tomster or your brand, and don't want to cause any long term problems. I asked for the Tomster in August simply to help identify Ember Twiddle as an official Ember subproject and to promote its use. I respect your team's decision not to permit its use.

My concern is that if a certain feature is released in the near future, we will have a very large opportunity in the short term to promote Ember Twiddle. I feel that some kind of promotional material and branding, including a logo, would be appropriate.

I think that if I just come up with something myself without your team's approval it will not be as good either for Ember Twiddle's future as an Ember subproject or our working relationship. I'd like to work with you, but please understand that where I'm coming from, I'd prefer to do it quickly. I humbly apologize if that makes things difficult, especially now that EmberConf is coming soon.

As to the website's domain, I would like a redirect from twiddle.emberjs.com, and like even more some placement on the emberjs.com website. However, I think enough time has passed where people associate the ember-twiddle.com domain with the project that changing it at this point may require considerable thought.

cc @wycats @joostdevries @alexspeller

**Comment by rwjblue** [comment]
> What do people think about having the project (eventually) live at twiddle.emberjs.com?

I do not believe we should do this for security reasons. For example, this is why Github pages was moved out from under the *.github.com domain ([see here](https://github.com/blog/1452-new-github-pages-domain-github-io)).

**Comment by stefanpenner** [comment]
@rwjblue ya agreed.

I would love to see more ember-twiddle mentions on the main site, it is an amazing and invaluable resource for all ember developers.

**Comment by acorncom** [comment]
@stefanpenner Any suggestions on where we should put the Ember Twiddle stuff?  I've been planning on updating the Community page with a link to it in the near future, but if you're noticing other places, I'm all ears ;-)

**Comment by stefanpenner** [comment]
Early thoughts, so bare with me but I wonder if it should also be prominent and on the main page. "Try ember in browser". 

I know I often jump to http://babeljs.io/repl for quick experiments, it is quite valuable so they have made it a primary navigation item.  I am unsure if we should do exactly that, but making it seem like part of the first-class tool chain seems good.

https://www.rust-lang.org/ even has a repl on the main page. (Likely a more long term goal, but our main-page likely also needs a refresh at some point)

**Comment by sheriffderek** [comment]
@stefanpenner Twiddle examples would be a major benefit on the official site and eventually possibly in the guides - but I wanted to reference #355 @Gaurav0 for this reason. If it were to be a user's first experience, the structure would need to leave no room for confusion about the conventions. I love twiddle - but as a intermediate Ember user, I still get tripped up with it's unique structure sometimes. It would be great to see that gap close.

**Comment by acorncom** [comment]
We'd love to use the Twiddles in the guides.  As Twiddle works right now, it's not really a good fit (due to the headaches surrounding ownership and updating code in the future).  We used to have JSBin snippets and ended up removing all of them for that reason (background info available here: https://github.com/emberjs/guides/issues/483).  It also gets worse with versioning ;-)

One example on the homepage though would probably be doable ...

**Comment by Gaurav0** [comment]
@acorncom Support for loading earlier revisions of a gist is now available, albeit not in UI. Forking and copying works. You can do a pull request from gist.github.com website. Please raise any issues remaining that still prevent you from using it and I will do my best to help.

**Comment by sheriffderek** [comment]
I like to have the guides open while I'm tinkering with ideas in twiddle. The attempt to conform to defacto Ember visual standards with regards to headers and UI and favicon is admirable, but not always useful in a practical sense. At a glance, I don't know which tab contains what. Right now, I just note which one is not retina quality - and I know it's twiddle. Favicons are neat little cake decorations, but they also serve as identifiers.

<img width="645" alt="screen shot 2016-04-04 at 5 29 45 pm" src="https://cloud.githubusercontent.com/assets/2066617/14296010/fc04923a-fb2c-11e5-8a83-7c3fe82c656a.png">

I don't have a unique solution, but the favicon (_dare I say all design_) should be a unique _spin_ on the ember 'look' and not just matching. Even if it's just black / or an inverse with an orange e.

**Comment by alexspeller** [comment]
We are basically stalled on this waiting for tilde to figure out what is allowed and what isn't in terms of branding. It doesn't make sense for ember-twiddle to make up its own inconsistent branding when it is an official ember project, so there's not really much we can do until tilde decide on what their plans are for subproject branding.
